front50-gcs/front50-gcs.gradle

```diff
@@ -19,6 +19,7 @@ dependencies {
   compile 'io.reactivex:rxjava:1.0.10'
 
   spinnaker.group('google')
+  compile spinnaker.dependency('clouddriverGoogleCommon')
   compile spinnaker.dependency("korkSecurity")
 
   /* spinnaker.group('googleStorage') */
```

Comment:
Turns out that dependency `googleStorage` is already in the `google` group, which is included just above. So I've dropped the line entirely.